src/pilot/install-director.sh

```diff
@@ -193,6 +193,11 @@ echo
 if [ -n "${overcloud_nodes_pwd}" ]; then
     echo "# Setting overcloud nodes password"
     virt-customize -a overcloud-full.qcow2 --root-password password:$overcloud_nodes_pwd
+    if [ $? -ne 0 ]; then
```

Comment:
If you specified an overcloud password to be set, this check will pass even tho the customize image script might have failed.